### DISCUSSION TITLE
Is it possible to manually create modpacks?

### DISCUSSION THREAD

**Discussion by FieryMewtwo** [SKCraft/Launcher#449] [I need help!]
Like, without the GUI? At all?

**Comment by hedgehog1029** [comment]
The creator tools internally makes use of the package builder, which has a CLI: https://github.com/SKCraft/Launcher/wiki/Command-Line-Tools

**Comment by FieryMewtwo** [comment]
Ok

**Comment by FieryMewtwo** [comment]
Wait, what about the first part where you select a version, etc.?

**Comment by FieryMewtwo** [comment]
never mind
